appengine/genetics/js/cage.js

```diff
@@ -31,34 +31,57 @@ goog.require('Genetics.Mouse');
 /**
  * The maximum population in a cage.
  * @type {number}
+ * @const
  */
-Genetics.Cage.MAX_POPULATION = 10;
+Genetics.Cage.MAX_POPULATION = 50;
 
 /**
  * The number of mice added to the cage for each player at the start of the
  * game. Should be less than MAX_POPULATION / number of players.
  * @type {number}
+ * @const
  */
 Genetics.Cage.START_MICE_PER_PLAYER = 2;
 
 /**
+ * Number of milliseconds between update calls.
+ * @type {number}
+ * @const
+ */
+Genetics.Cage.UPDATE_DELAY_MSC = 50;
+
+/**
  * Whether the simulation is running without a visualization.
  * @type {boolean}
  */
 Genetics.Cage.HEADLESS = false;
 
 /**
- * Mapping of player ID to array with player name and code.
- * @type {!Object.<number, Array.<string>>}
+ * Whether the simulation is stopped.
+ * @type {boolean}
```

Comment:
Yes, players is const, I'll do a run through and try to catch other arrays that are not reassigned. According to the style guide:

`/**
 * Map of URL to response string.
 * @const
 */
MyClass.fetchedUrlCache_ = new goog.structs.Map();`

> In this case, the reference can never be overwritten, but value itself is mutable (and thus in camelCase, not ALL_CAPS).